src/instructlab/model/serve.py

```diff
@@ -129,6 +121,14 @@ def serve(
         )
     elif backend == backends.VLLM:
         # Instantiate the vllm server
+        if ctx.args:
+            # extra click arguments after "--"
+            vllm_args = ctx.args
+        elif ctx.obj.config.serve.vllm.vllm_args:
```

Comment:
if the options are still in the cfg, then I don't think this impacts #1634. I honestly like this new way of dealing w/ the flags.